apps/ledger-live-mobile/src/screens/Manager/Benchmarking.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+import { useAppInstallProgress } from "@ledgerhq/live-common/apps/react";
+import Config from "react-native-config";
+
+const Wrapper = styled(Flex).attrs({
+  bg: "palette.background.default",
+  p: 4,
+  mt: 4,
+  borderWidth: 1,
+  borderRadius: 4,
+  borderColor: "neutral.c40",
+})``;
+
+const Status = styled(Flex).attrs(p => ({
+  bg: p.active ? "success.c100" : "warning.c100",
```

Comment:
I'm not the expert on styled + Flex, so not sure if it's a real problem. But TS is complaining that:

```
Property 'active' does not exist on type 'ThemedStyledProps<ViewProps & RefAttributes<View>, DefaultTheme>'.ts(2339)
```